chembl_structure_pipeline/test/test_standardizer.py

```diff
@@ -3186,3 +3186,92 @@ def testBondWedgingSurvivingSaltStripping(self):
         inchi1 = Chem.MolBlockToInchi(mb)
         inchi2 = Chem.MolBlockToInchi(smb)
         self.assertEqual(inchi1,inchi2)
+
+
+    def testParseMolblock(self):
+        ' Check parse_molbock function '
+        mb = '''
+  Mrv1810 07121910172D          
+
+  4  3  0  0  0  0            999 V2000
+   -2.5038    0.4060    0.0000 C   0  0  3  0  0  0  0  0  0  0  0  0
+   -2.5038    1.2310    0.0000 O   0  5  0  0  0  0  0  0  0  0  0  0
+   -3.2182   -0.0065    0.0000 N   0  3  0  0  0  0  0  0  0  0  0  0
+   -1.7893   -0.0065    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+  1  3  1  0  0  0  0
+  1  4  1  4  0  0  0
+M  CHG  2   2  -1   3   1
+M  END'''
+        mtt = standardizer.parse_molblock(mb, useRDKitChemistry=True)
+        assert type(mtt) is Chem.Mol
+        mtf = standardizer.parse_molblock(mb, useRDKitChemistry=False)
+        assert type(mtf) is Chem.Mol
+
+
+        mb = ''' wrong valence
+  Mrv1810 07121910172D          
+
+  4  3  0  0  0  0            999 V2000
+   -2.5038    0.4060    0.0000 C   0  0  3  0  0  0  0  0  0  0  0  0
+   -2.5038    1.2310    0.0000 O   0  5  0  0  0  0  0  0  0  0  0  0
+   -3.2182   -0.0065    0.0000 N   0  3  0  0  0  0  0  0  0  0  0  0
+   -1.7893   -0.0065    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+  1  2  1  0  0  0  0
+  1  3  1  0  0  0  0
+  1  4  1  4  0  0  0
+M  CHG  9   2  -1   3   1
+M  END'''
+
+        mft = standardizer.parse_molblock(mb, useRDKitChemistry=True)
+        assert type(mft) is type(None)
```

Comment:
here you could use `self.assertIsNone(mft)`

Reply:
sure, thanks for the patience :P I'm updating this and packing now.